Tomes/Tome.cs

```diff
@@ -23,69 +27,83 @@ public class Tome {
 		}
 
 		//
-		public static void Destroy(JToken data) {
-			switch (data.Type) {
+		public static void Destroy(JToken data)
+		{
+			switch (data.Type)
+			{
 				case JTokenType.Array:
-					(data as TomeArray).Destroy();
+					((TomeArray)data).Destroy();
 					break;
 				case JTokenType.Object:
-					(data as TomeObject).Destroy();
+					((TomeObject)data).Destroy();
 					break;
 				default:
-					(data as TomeValue).Destroy();
+					((TomeValue)data).Destroy();
 					break;
 			}
 		}
 
 		//
-		public static JToken PathValue(JToken value, JArray paths) {
-			foreach (JToken path in paths) {
+		public static JToken PathValue(JToken value, JArray paths)
+		{
```

Comment:
the question is should it explode with exception? as you shouldn't pass in null
or should it quietly return the value you fed it?
i think originally i wanted it to explode on null values as you shouldn't feed that in
perhaps null checks should go elsewhere?